homeassistant/components/deconz/__init__.py

```diff
@@ -58,28 +57,25 @@
     return True
 
 
-async def async_setup_entry(hass, entry):
+async def async_setup_entry(hass, config_entry):
     """Set up a deCONZ bridge for a config entry."""
     if DOMAIN in hass.data:
         _LOGGER.error(
             "Config entry failed since one deCONZ instance already exists")
         return False
-    result = await async_setup_deconz(hass, None, entry.data)
-    if result:
-        return True
-    return False
+    return await async_setup_deconz(hass, config_entry)
 
 
-async def async_setup_deconz(hass, config, deconz_config):
+async def async_setup_deconz(hass, config_entry):
     """Set up a deCONZ session.
 
     Load config, group, light and sensor data for server information.
     Start websocket for push notification of state changes from deCONZ.
     """
-    _LOGGER.debug("deCONZ config %s", deconz_config)
+    _LOGGER.debug("deCONZ config %s", config_entry.data)
```

Comment:
Writing auth credentials to the log

Reply:
There are a couple of places in the lib that also happens, but only when enabling debug. Another issue is that all communication is http only.